### PR TITLE
feat(assert): add asserts to check element

### PR DESCRIPTION
**Pull request by bdejacobet**
- [x] add some methods to check if element is present or not with a laps of time
- [x] add PHP unit tests

### CODE REVIEW

**Comment by bdejacobet** [comment]
Erf, there are already same methods in `src/ExtraWebAssertTrait.php` :/
which ones should we keep?

File: tests/ExtraWebAssertTraitTest.php [hunk 1]
```diff
@@ -79,10 +79,92 @@ public function testClickElement()
...
+     */
+    public function testSpinThrowsException()
+    {
+        $session = $this->createMock(Session::class);
+        $session->expects($this->once())->method('getDriver')->willReturn($this->createMock(DriverInterface::class));
+
+        $mock = $this->getExtraWebAssertMockWithoutProtectedMethods();
+        $mock->expects($this->once())->method('getSession')->willReturn($session);
+
+        $method = new \ReflectionMethod(get_class($mock), 'spin');
```
Comment (bdejacobet) [inline tests/ExtraWebAssertTraitTest.php:142]: Use `invokeMethod` of `tests/Traits/TestHelperTrait.php`

**Comment by mremi** [comment]
Let's merge both and keep `ExtraSessionTrait`

**Comment by bdejacobet** [comment]
Cancel! Changes added to https://github.com/ekino/behat-helpers/pull/6

**Pull request closed without merging**